pallets/ah-migrator/src/lib.rs

```diff
@@ -256,6 +273,23 @@ pub mod pallet {
 		type SendXcm: SendXcm;
 		/// Weight information for extrinsics in this pallet.
 		type AhWeightInfo: WeightInfo;
+		/// Asset Hub Treasury accounts migrating to the new treasury account address (same account
+		/// address that was used on the Relay Chain).
+		///
+		/// The asset list should not include the native asset.
+		type TreasuryAccounts: Get<(Self::AccountId, Vec<<Self::Assets as FungiblesInspect<Self::AccountId>>::AssetId>)>;
+		/// Convert the Relay Chain Treasury Spend (AssetKind, Beneficiary) parameters to the
+		/// Asset Hub (AssetKind, Beneficiary) parameters.
+		type RcToAhTreasurySpend: Convert<
```

Comment:
There is `TryConvert` for fallible ones.

Reply:
in this case I would have to clone the arguments to return them in case of the error, this is why convert